linkerd/core/src/test/scala/io/buoyant/linkerd/TestProtocol.scala

```diff
@@ -141,12 +141,10 @@ object TestProtocol {
   }
 
   object Plain extends TestProtocol("plain") {
-    val configClass = Parser.jClass[PlainConfig]
-    val configId = "plain"
+    val configClass = classOf[PlainConfig]
   }
 
   object Fancy extends TestProtocol("fancy") {
-    val configClass = Parser.jClass[FancyConfig]
-    val configId = "fancy"
+    val configClass = classOf[FancyConfig]
```

Comment:
I noticed that if we want to match the naming pattern, FancyConfig should be Fancy, Fancy should probably be FancyProtocol. Same with Plain